core/table.ts

```diff
@@ -355,29 +356,35 @@ export class Table {
     const incrementalContext = new TableContext(this, true);
 
     this.proto.query = context.apply(this.contextableQuery);
+
+    const createOps = (
```

Comment:
nit: can this instead be a private method?

Reply:
Sure, does indeed look nicer.